storage/id_alloc.go

```diff
@@ -100,14 +100,13 @@ func (ia *idAllocator) start() {
 					res client.KeyValue
 				)
 				for r := retry.Start(idAllocationRetryOpts); r.Next(); {
-					task := ia.stopper.StartTask()
-					if !task.Ok() {
+					var idKey interface{}
```

Comment:
This variable should be a `proto.Key` (or else the type assertion on line 105 is redundant).